Assets/Scripts/UI/CameraMode.cs

```diff
@@ -0,0 +1,6 @@
+﻿public enum CameraMode
```

Comment:
Maybe add one for player titans? That one shouldn't be saved as a preferred camera mode, though, so maybe not.